packages/freezed/lib/src/templates/properties.dart

```diff
@@ -106,13 +136,16 @@ class Property {
         body: body,
       );
 
-  Setter get unimplementedSetter => Setter(
-        name: name,
-        type: type,
-        decorators: decorators,
-        doc: doc,
-        body: ' => throw $privConstUsedErrorVarName;',
-      );
+  Setter? get unimplementedSetter =>
```

Comment:
I don't like that this is nullable. It doesn't seem like the correct place to put logic here.